dbt/adapters/bigquery/impl.py

```diff
@@ -280,7 +280,8 @@ def list_relations_without_caching(
         # the implementation of list_relations for other adapters
         try:
             return [self._bq_table_to_relation(table) for table in all_tables]  # type: ignore[misc]
-        except google.api_core.exceptions.NotFound:
+        except google.api_core.exceptions.NotFound as exc:
+            logger.debug("list_relations_without_caching error: {}".format(str(exc)))
```

Comment:
This shouldn't be considered an "error" per se 
```suggestion
            logger.debug("list_relations_without_caching table not found: {}".format(str(exc)))
```

Reply:
@colin-rogers-dbt good call about the update in wording 🧠